models/webhook/webhook.go

```diff
@@ -192,6 +194,9 @@ type Webhook struct {
 	Meta            string     `xorm:"TEXT"` // store hook-specific attributes
 	LastStatus      HookStatus // Last delivery status
 
+	// HeaderAuthorizationEncrypted should be accessed using HeaderAuthorization() and SetHeaderAuthorization()
+	HeaderAuthorizationEncrypted string `xorm:"TEXT"` // TODO: migration
```

Comment:
For migration you would copy paste this current struct(including the new field) into a new migration file(atm this would be `models/migrations/v226.go`). Look at models/migrations/v219.go at how which code you should use in that file in order to "sync it". And add some code to `models/migrations/migrations.go` that reference the function in v226.go

Reply:
Thanks for the explanation and examples!

I added 2 migrations: one adding the column and the other converting the existing matrix webhooks (with a test).

Reply:
Could the comment `TODO` be removed now?

Reply:
I (finally) removed the comment